Package.swift

```diff
@@ -5,6 +5,12 @@ import PackageDescription
 
 let package = Package(
     name: "swift-concurrent-sequence",
+    platforms: [
+        .macOS(.v12),
+        .iOS(.v15),
+        .tvOS(.v15),
+        .watchOS(.v8),
+    ],
```

Comment:
I plan on updating the README once this PR lands as this PR is already too large. I'll be sure to include the minimum requirements information when I do this.